filebeat/harvester/reader/line.go

```diff
@@ -16,13 +16,12 @@ type Line struct {
 	rawInput   io.Reader
 	codec      encoding.Encoding
 	bufferSize int
-
-	nl        []byte
-	inBuffer  *streambuf.Buffer
-	outBuffer *streambuf.Buffer
-	inOffset  int // input buffer read offset
-	byteCount int // number of bytes decoded from input buffer into output buffer
-	decoder   transform.Transformer
+	nl         []byte
+	inBuffer   *streambuf.Buffer
+	outBuffer  *streambuf.Buffer
+	inOffset   int // input buffer read offset
+	byteCount  int // number of bytes decoded from input buffer into output buffer
+	decoder    transform.Transformer
```

Comment:
Here I think the mess is also related to https://github.com/elastic/beats/pull/2089 and some tricks ;-) We should probably clean this up in combination with https://github.com/elastic/beats/pull/2089